build.sh

```diff
@@ -14,14 +14,12 @@ export CMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER:-mpif90}
 export NETCDF=${NETCDF:?"Please set NETCDF environment variable"}
 export ESMFMKFILE=${ESMFMKFILE:?"Please set ESMFMKFILE environment variable"}
 
-BUILD_DIR=${MYDIR}/build
-rm -rf ${BUILD_DIR}
-mkdir ${BUILD_DIR}
+BUILD_DIR=${BUILD_DIR:-${UFS_MODEL_DIR}/build}
+mkdir -p ${BUILD_DIR}
 
-CCPP_SUITES="${CCPP_SUITES:-FV3_GFS_v15p2}"
-CMAKE_FLAGS+=" -DCCPP_SUITES=${CCPP_SUITES} -DNETCDF_DIR=${NETCDF}"
+[[ -n "${CCPP_SUITES:-""}" ]] && CMAKE_FLAGS+=" -DCCPP_SUITES=${CCPP_SUITES}"
+CMAKE_FLAGS+=" -DNETCDF_DIR=${NETCDF}"
 
 cd ${BUILD_DIR}
-cmake .. ${CMAKE_FLAGS}
+cmake ${UFS_MODEL_DIR} ${CMAKE_FLAGS}
 make -j ${BUILD_JOBS:-4}
```

Comment:
can we add verbose option, default off?
e.g.
`make -j ${BUILD_JOBS:-4} VERBOSE=${BUILD_VERBOSE:-0}`

Reply:
Good idea. Let's add this next time, to avoid rerunning all the tests. And BUILD_VERBOSE should be set/exported in compile.sh.

Reply:
> Good idea. Let's add this next time, to avoid rerunning all the tests. And BUILD_VERBOSE should be set/exported in compile.sh.

This is a great idea. - next commit!